README.md

```diff
@@ -2,82 +2,44 @@
 
-Download helm from: <https://github.com/helm/helm/releases>, unpack and move
-binaries to most convenient place for you.
+Repository of Wharf's [Helm](https://helm.sh/) charts. Currently hosting:
 
-For more information surf to install guide: <https://helm.sh/docs/intro/install/>
+- [`wharf-helm`](charts/wharf-helm/README.md):
+  Deploy Wharf to Kubernetes using Helm
 
-On windows add helm path to PATH in environment variables.
-
-Do a test render into stdout using helm template:
-
-```sh
-# helm v2
-helm template wharf-helm
-# helm v3
-helm template wharf-helm --generate-name
-```
-
-Test for common errors using `helm lint`:
+To start using, add the Wharf Helm chart repository:
 
 ```sh
-helm lint wharf-helm
+helm repo add iver-wharf https://iver-wharf.github.io/wharf-helm
 ```
 
-To install the chart on kubernetes cluster use command:
-
-Locally
+## Generating `charts/*/README.md`
 
-```sh
-helm install wharf --namespace wharf ./wharf
-```
-
-Repo
+The `README.md` files in each chart is generated using
+[norwoodj/helm-docs](https://github.com/norwoodj/helm-docs).
 
-```sh
-helm repo add wharf https://<url_when_public>/wharf-charts
-helm repo update
-helm install wharf --namespace wharf wharf/wharf
-```
+1. Install `helm-docs`. Installation instructions can be found at:
+   <https://github.com/norwoodj/helm-docs#installation>
 
-Use -f values.yaml and point it to a modified values.yaml file to overrride
-default values.
+   If you have Go installed, you may run:
 
-## Publishing
-
-1. Add a merge request to set the date of the deployment in the chart's
-   "Changes" section of the `wharf-helm/README.md` file.
-
-   ```diff
-   diff --git a/wharf-helm/README.md b/wharf-helm/README.md
-   index 050b39f..68ff38f 100644
-   --- a/wharf-helm/README.md
-   +++ b/wharf-helm/README.md
-   @@ -2,7 +2,7 @@
-
-    ## Changes
-
-    -### 1.0.0 (WIP)
-    +### 1.0.0 (2021-02-12)
-
-     - BREAKING: Changed `.Values.postgres` to `.Values.postgresKubeDB`
+   ```sh
+   go get -u github.com/norwoodj/helm-docs/cmd/helm-docs
    ```
 
-2. Doublecheck that the version in `Chart.yaml` is correct for the chart.
-
-3. Run a build in Wharf to build the stage that packages the chart.
-
-4. Tag the latest commit on `master` with the version, prefixed with "v":
+2. Run `helm-docs`, preferrably before you create your pull requests:
 
    ```sh
-   git checkout master
+   helm-docs
+   ```
 
-   git pull
+3. Commit the changes on the `README.md` files.
 
-   git tag v1.0.0 -m "wharf-helm v1.0.0"
-   ```
+## Publishing
 
-5. Done!
+The charts in this repository are published automatically using
+[helm/chart-releaser](https://github.com/helm/chart-releaser). No further
+action is required except getting your change into the `master` branch.
```

Comment:
Q: Do we want to use this auto-deployment feature? Or do we want to do manual deployments by using the `cr` tool locally instead?

I'm torn between using auto deploy for convinience, but using manual deployment for greater control and would allow us to stick to the same versioning scheme we're using in the other repos where it's a manual task to release, as well as allowing us to better group changes into a single version

Reply:
I've thought about it and I'm changing to requireing manual deployment

Reply:
Im a big fan of automation but im have never used the cr tool nor do i know the scope of the work required for a redeployment.

Reply:
I played around a little with the `cr` tool locally and it's not that user friendly. Think relying on https://github.com/helm/chart-releaser-action is better than dealing with that storm.

I tested out the GitHub action over here: https://github.com/jilleJr/helm-test and it was a breeze in comparison. Really plug and play, with the loss of control when it comes to versioning. We can perhaps use release branches instead for this repo, like `release/v1.0.0` or something

Reply:
Resolving this